settings/remark-language-server.vim

```diff
@@ -0,0 +1,14 @@
+augroup vim_lsp_settings_remark_language_server
+  au!
+  LspRegisterServer {
+      \ 'name': 'remark-language-server',
+      \ 'cmd': {server_info->lsp_settings#get('remark-language-server', 'cmd', [lsp_settings#exec_path('typescript-language-server')]+lsp_settings#get('remark-language-server', 'args', ['--stdio']))},
```

Comment:
This `typescript-language-server` is correctly?

Reply:
Now I got this working. Could you please fix `s/typescript/remark/` ?
![image](https://user-images.githubusercontent.com/10111/147385407-acf4eee8-ad32-42da-9399-39e41ce977a6.png)

Reply:
Awesome! Thanks for helping troubleshoot